src/HomeWork_4/HomeWork_4.ts

```diff
@@ -3,37 +3,86 @@
 // які не можна змінювати після створення.У Square і Rectangle зі свого боку є
 // ще додатковий метод print, який виводить рядок із формулою розрахунку площі
 
-interface IFigure {
```

Comment:
You can incorporate this interface into an abstract class, which also makes sense.